packages/elements/src/lib/browser.ts

```diff
@@ -1,7 +1,10 @@
+import { isServer } from 'lit';
+
 /**
  * Test if localStorage exists and is enabled
  */
 export function isLocalStorageAvailable() {
+  if (isServer) return false;
```

Comment:
Because this is a static boolean based on [export conditionals](https://nodejs.org/api/packages.html#conditional-exports) this is really hard to test :(. Because our tests run in the browser this will always just be `false`